client/state/login/test/selectors.js

```diff
@@ -45,15 +50,15 @@ describe( 'selectors', () => {
 				},
 			} );
 
-			expect( id ).to.equal( 123456 );
+			expect( id ).toStrictEqual( 123456 );
```

Comment:
Nit: this check can be a simple `.toBe`. [`toStrictEqual`](https://jestjs.io/docs/en/expect.html#tostrictequalvalue) is an extended version of deep equality check, useful for checking class instances. I don't remember ever seeing it in practice. We'd probably use it more if there was more OOP and prototypes in our code.

Reply:
I should probably have read the Jest documentation a bit closer. Thanks, done!